docs/source/customization.rst

```diff
@@ -0,0 +1,68 @@
+
+=====================
+ Customizing the API
+=====================
+
+Content serialization
+=====================
+
+Dexterity fields
+----------------
+
+The API automatically converts all field values to JSON compatible
+data, whenever possible.
+However, you might use fields which store data which cannot automatically
+be converted or you might want to customize the representation of certain
+fields.
+
+For extending or changing the serializing of certain dexterity fields you
+need to register an ``IFieldSerializer``-adapter.
+
+Example:
+
+.. code:: python
+
+    from plone.customfield.interfaces import ICustomField
+    from plone.dexterity.interfaces import IDexterityContent
+    from plone.restapi.interfaces import IFieldSerializer
+    from plone.restapi.serializer.converters import json_compatible
+    from plone.restapi.serializer.dxfields import DefaultFieldSerializer
+    from zope.component import adapter
+    from zope.interface import Interface
+    from zope.interface import implementer
+
+
+    @adapter(ICustomField, IDexterityContent, Interface)
+    @implementer(IFieldSerializer)
+    class CustomFieldSerializer(DefaultFieldSerializer):
+
+        def __call__(self):
+            value = self.get_value()
+            if value is not None:
+                value = value.output()
```

Comment:
Maybe insert a comment above this line `# Do something with value` to distinguish it a bit from the rest of the code, which is more or less boilerplate.